src/pages/registration/resource_type_tab/components/JournalField.tsx

```diff
@@ -147,6 +152,24 @@ export const JournalField = ({ confirmedContextType, unconfirmedContextType }: J
             renderOption={(props, option, state) => (
               <PublicationChannelOption key={option.id} props={props} option={option} state={state} />
             )}
+            PaperComponent={({ children, ...props }) => {
```

Comment:
Har prøvd myyye frem og tilbake for å løse dette med `ListboxComponent`-prop i stedet, siden vi da kunne lagt "Vis mer" som siste element i lista, i motsetning til nå hvor knappen alltid vil vises nederst, selv om man har scrollet opp og ikke kan se siste element... Liker ikke denne løsningen men fikk det rett og slett ikke til med `ListboxComponent`, så da tror jeg det må bli sånn om ingen andre har noen gode forslag :/

I så fall blir det å flytte dette til en egen komponent, som også skal brukes av `PublisherField` og `SeriesField`, men som også kan brukes for alle andre Autocomplete-søk, feks prosjekt om det er ønskelig i fremtiden.

Reply:
Trøbbel med PaperComponent også, ved at popul lukkes når man trykker utenfor listeelementer, OG at vis flere-knappen alltid vil vises, også når man ikke har scrollet til siste element.

Har en nesten-løsning med `ListboxComponent`, hvor eneste problemet er at hver option ikke får en hover-effekt, uten at jeg skjønner hvorfor:
```ListboxComponent={({ children, ...props }) => (
  <ul {...props}>
    {children}
    {!!journalOptionsQuery.data?.totalHits && journalOptionsQuery.data.totalHits > searchSize && (
      <li>
        <LoadingButton
          sx={{ m: '0.5rem' }}
          endIcon={<ExpandMore />}
          loading={journalOptionsQuery.isFetching && !journalOptionsQuery.isPending}
          onClick={() => setSearchSize(searchSize + defaultSearchSize)}>
          {t('common.show_more')}
        </LoadingButton>
      </li>
    )}
  </ul>
)}```

Reply:
Tror jeg fant en slags løsning nå. Alt ser ut til å virke, men må nok gjøre en jobb med å fikse typer, så spørs det hvor godt dette skalerer når det skal inn andre plasser også..
```
<Autocomplete
  (...)
  ListboxComponent={CustomListboxComponent}
  ListboxProps={
    {
      hasMoreHits: true,
      onShowMoreHits: () => setSearchSize(searchSize + defaultSearchSize),
      isLoadingMoreHits: journalOptionsQuery.isFetching && !journalOptionsQuery.isPending,
    } as any
  }
/>

interface CustomProps {
  hasMoreHits?: boolean;
  onShowMoreHits?: () => void;
  isLoadingMoreHits?: boolean;
}

const CustomListboxComponent = forwardRef<HTMLUListElement, React.HTMLProps<HTMLUListElement> & CustomProps>(
  function CustomListboxComponent(props, ref) {
    const { children, hasMoreHits, onShowMoreHits, isLoadingMoreHits, ...other } = props;
    const { t } = useTranslation();

    return (
      <ul {...other} ref={ref}>
        {children}
        {hasMoreHits && (
          <LoadingButton
            sx={{ m: '0.5rem' }}
            endIcon={<ExpandMore />}
            loading={isLoadingMoreHits}
            onClick={onShowMoreHits}>
            {t('common.show_more')}
          </LoadingButton>
        )}
      </ul>
    );
  }
);
```